From 60b2476f08ec942dccaa6bda4228c2d51cce4b05 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Wed, 20 Aug 2014 18:25:29 +0300
Subject: [PATCH] ARM: dts: hummingboard: gpio-ir on gpio 3,5

HummingBoard since rev 2.0 (few developers had it) and the production one
starting rev 3.0 uses gpio 3,5 (EIM_DA5 pad) as the gpio infra red receiver
input.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/boot/dts/imx6qdl-hummingboard.dtsi |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-hummingboard.dtsi b/arch/arm/boot/dts/imx6qdl-hummingboard.dtsi
index f725dfa..09c293d 100644
--- a/arch/arm/boot/dts/imx6qdl-hummingboard.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-hummingboard.dtsi
@@ -11,9 +11,9 @@
 
 	ir_recv: ir-receiver {
 		compatible = "gpio-ir-receiver";
-		gpios = <&gpio1 2 1>;
+		gpios = <&gpio3 5 1>;
 		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_hummingboard_gpio1_2>;
+		pinctrl-0 = <&pinctrl_hummingboard_gpio3_5>;
 	};
 
 	regulators {
@@ -189,9 +189,9 @@
 			>;
 		};
 
-		pinctrl_hummingboard_gpio1_2: hummingboard-gpio1_2 {
+		pinctrl_hummingboard_gpio3_5: hummingboard-gpio3_5 {
 			fsl,pins = <
-				MX6QDL_PAD_GPIO_2__GPIO1_IO02 0x80000000
+				MX6QDL_PAD_EIM_DA5__GPIO3_IO05 0x80000000
 			>;
 		};
 
-- 
1.7.9.5

